tests/src/test/scala/cats/tests/FoldableSuite.scala

```diff
@@ -151,7 +151,7 @@ abstract class FoldableSuite[F[_]: Foldable](name: String)(implicit ArbFInt: Arb
       fa.collectFold(pf) should ===(fa.toList.collect(pf).fold(m.empty)(m.combine))
 
       def g(a: String): Option[String] = Some(a).filter(f)
-      fa.collectSomeFold(g) should ===(fa.toList.filter(f).fold(m.empty)(m.combine))
+      fa.collectFoldSome(g) should ===(fa.toList.filter(f).fold(m.empty)(m.combine))
```

Comment:
I'd intentionally left this testing the deprecated version for the sake of coverage and safety (I got bitten pretty badly by an untested deprecated method in Circe 0.12.0). I think we're probably okay changing this instance now, but we should have a policy for what to do in these cases.

Reply:
I didn't know it, happy to revert. It's not relevant for the issue. I will comment there.

Reply:
Thanks! (I should have done that already).

Reply:
thanks for the review.